packages/scxa-cell-type-wheel/src/CellTypeWheel.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import HighchartsReact from 'highcharts-react-official'
+import Highcharts from 'highcharts'
+import highchartsSunburst from 'highcharts/modules/sunburst'
+
+highchartsSunburst(Highcharts)
+
+// SUPER HACK!
+// https://github.com/highcharts/highcharts/issues/4994
+window.Highcharts = Highcharts
+
+class CellTypeWheel extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      // To avoid unnecessary update keep all options in the state.
```

Comment:
Sorry I did not get this comment, whey do we need to set those options as state? I think we could utilize a stateless component here.

Reply:
Sorry, this was a bit of a distracting comment. I copy-pasted the template from https://github.com/highcharts/highcharts-react#optimal-way-to-update, and it’s actually an explanation of the Highcharts wrapper, not about our code. I’ve removed it.

Reply:
Yes, but still we do not need a stateful component here as far as I can see.

Reply:
In principle we don’t, but observer that ultimately the `render` is called by `HighchartsReact` and you don’t know how it manages things internally. Highcharts official documentation states that:
> A good practice is to keep all chart options in the state. When `setState` is called, the options are overwritten and only the new ones are passed to the `chart.update` method.

I say, let’s go with that and we’ll have fewer surprises down the road. Maybe everything’s fine with a purely functional component, but I prefer to stick to Highcharts docs so if we get some unexepected behaviour we can report it (and get an upstream fix) sooner.

Reply:
Okay, no problem. I did not know that Highcharts suggests in this way. I think functional component is lighter than stateful ones.